metadata-ingestion/src/datahub/ingestion/source/looker/looker_config.py

```diff
@@ -81,6 +81,7 @@ class NamingPatternMapping:
     project: str
     model: str
     name: str
+    file_path: Optional[str]
```

Comment:
file_path up as an option in explore_naming_pattern and explore_browse_pattern as well whereas we don't need them there. See - https://docs-website-1dask8vf5-acryldata.vercel.app/docs/next/generated/ingestion/sources/looker

Probably create a subclass `ViewNamingPatternMapping` with `file_path:str` and remove it from parent class ?

Reply:
done

Reply:
not able to see file_path in docs against view naming pattern and view_browse_path, can you please check ?

Reply:
It is doc generation issue, need help from @hsheth2